### PR TITLE
Add sponsor logos

### PR DESCRIPTION
**Pull request by skorbut**
Add all confirmed sponsors to website and add dedicated sponsor page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,6 +23,22 @@
       </div>
     </header>
 
+    <section id="sponsor-gold" class="index-sponsors">
```
Comment (stefan-evernest) [inline index.html]: proposal: I'd put the first sponso section under the about section to not confuse visitors with that gigantic logo

File: style.css [hunk 2]
```diff
@@ -117,6 +117,50 @@ section {
...
+  margin: 2rem 0;
+}
+
+.sponsor__logo__gold img {
+  width: 450px;
+  max-width: 90%;
+}
+
+.sponsor__logo__silver img {
+  width: 375px;
```
Comment (stefan-evernest) [inline style.css:152]: issue: this produces a horizontal scrollbar on iPhone SE due to limited screen size. Make sure to always have a `max-width` in place (as in gold sponsors)

**Review by stefan-evernest — CHANGES_REQUESTED** [review]

File: sponsors.html
Comment (stefan-evernest) [inline sponsors.html:1]: issue: there should be a back button and / or a logo on header with link to the index file to make navigation easier

**Review by emrox — APPROVED** [review]

File: style.css (hunk 2 above)
Comment (emrox) [reply style.css:152]: https://github.com/rubyunconfeu/2024-website/pull/5/commits/f27166ae25ffd341a9dce39200dfceab1683eb7b

**Pull request merged**